draft-ietf-cose-bls-key-representations.md

```diff
@@ -200,6 +200,49 @@ E'(GF(p^8))
 
 {backmatter}
 
+# Appendix
+
+The following examples showcase JWKs for both the G1 and G2 subgroups of the Bls12381 curve. Note, the examples also include the corresponding private key, expressed through the inclusion of the “d” parameter.
+
+An example JWK for the Bls12381 curve where the public key is in the G1 subgroup.
+
+```
+{
+  "kty": "OKP",
+  "crv": "Bls12381G1",
+  "d": "Mt_OyD9IAsYvobHJ9NCipm6-G7zAu28FCc-saRnXhjQ",
+  "x": "iXmOmxttBniHSpyoq-vBr82BexrqG7WDTsxCY4ngUOERVxwpwUT7yKqSKqJeIr7J"
+}
+Another example of a different JWK for the Bls12381 curve where the public key is in the G1 subgroup.
+```
+{
```

Comment:
```suggestion
```

Another example of a different JWK for the Bls12381 curve where the public key is in the G1 subgroup.
```
{
```